test/client-test.js

```diff
@@ -175,5 +179,232 @@ describe('Client', () => {
 
       await this.fhirClient.read({ resourceType: 'Patient', id: 'test-access-token' });
     });
+
+    describe('#create', () => {
+      it('returns a successful operation outcome', async function () {
+        const newPatient = {
+          resourceType: 'Patient',
+          active: true,
+          name: [{ use: 'official', family: ['Coleman'], given: ['Lisa', 'P.'] }],
+          gender: 'female',
+          birthDate: '1948-04-14',
+        };
+
+        nock(this.baseUrl)
+          .matchHeader('accept', 'application/json+fhir')
+          .post('/Patient')
+          .reply(201, () => readStreamFor('patient-created.json'));
+
+        const response = await this.fhirClient.create({ resourceType: 'Patient', body: newPatient });
+
+        expect(response.resourceType).to.deep.equal('OperationOutcome');
+        expect(response.issue[0].diagnostics).to.have.string('Successfully created resource');
+      });
+
+      it('throws an error if the resource is not supported', async function () {
+        const newRecord = {
+          resourceType: 'Foo',
+          name: [{ use: 'official', family: ['Coleman'], given: ['Lisa', 'P.'] }],
+        };
+
+        nock(this.baseUrl)
+          .matchHeader('accept', 'application/json+fhir')
+          .post('/Foo')
+          .reply(400, () => readStreamFor('unknown-resource.json'));
+
+        let response;
+        try {
+          response = await this.fhirClient.create({ resourceType: 'Foo', body: newRecord });
+        } catch (error) {
+          expect(error.response.status).to.eq(400);
+          expect(error.response.data.resourceType).to.deep.equal('OperationOutcome');
```

Comment:
I think we can just do `to.equal('OperationOutcome')` here and throughout tests-- usually just need deep equality for objects